doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+   :widths: 1 2 
+   :header-rows: 1
+   :class: longtable
+
+   *  -  Implementation 
+      -  Mitigations
+
+      
+   *  -  `DM.PROTECTED`
+      -  `M.FullyQualifiedNames`
+         `M.GlitchDetection`
+         `M.ReadAfterWrite`
+         `M.UseSecurepartitions`
+
+   *  -  `DM.EXPOSED`
+      -  `M.AntiRollback`
+         `M.Encrypt`
+         `M.FullyQualifiedNames`
+         `M.GlitchDetection`
+         `M.MAC`
+         `M.ReadAfterWrite`
+         `M.UseSecurepartitions`
+         
+   *  -  `DM.AUTHORISED`
+      -  `M.AuthenticateEndpoints`
+         `M.FullyQualifiedNames`
+         `M.GlitchDetection`
+         `M.ReadAfterWrite`
+         `M.Replay Protection`
+         `M.UniqueKeys`
+         `M.UseSecurepartitions`
+         `M.VerifyReplies`
+         
+   *  -  `DM.SECURE_CHANNEL`
+      -  `M.AuthenticateEndpoints`
+         `M.FullyQualifiedNames`
+         `M.GlitchDetection`
+         `M.PRoT rooted Secure Channel`
+         `M.ReadAfterWrite`
+         `M.Replay Protection`
+         `M.UniqueKeys` 
+         `M.UseSecurepartitions`
+         `M.VerifyReplies`
+         
+
+In implementation `DM.PROTECTED`, `DM.SECURE_CHANNEL`, the stored data can be implicitly trusted, and therefore it is not required to be encrypted or authenticated. There is also no more secure location to store verification data. However, it is possible for the data to be accidentally corrupted, therefore standard engineering practice to guard against this, for example the use of error correcting codes, should be used. 
+
+In implementation `DM.EXPOSED`, the data can be read or modified by an attacker, therefore the Storage Service must provide confidentiality, integrity, and authenticity by cryptographic means. The keys used to do this must be stored securely. This could be a key derived from the HUK, or separately stored in fuse in a location only readable from the :term:`PRoT`. 
+
+As the attacker can always read and modify the stored data, even if they cannot decrypt the data. They can attempt to subvert a change by resetting the Storage Medium to a prior state. To detect this, the Storage Service needs to have some means of authenticating that it is reading the most recent state. This implies some form of authentication data stored in a location the attacker cannot modify.
+
+In implementation `DM.AUTHORISED`, the data can be observed, even if it cannot be modified. Therefore, data stored does need to be encrypted for confidentiality. However, provided the authentication protocol is strong, and prevents replay, it should not be possible for an attacker to modify the stored data. As the store applies a MAC to each reply, the Storage Service does not need to apply extra integrity. 
+
+In implementation `DM.SECURE_CHANNEL` provided the secure channel is rooted within the :term:`PRoT`, the data transferred cannot be observed, and any modification will be detected. Therefore, no further encryption is needed for confidentiality or integrity. 
+

```

Comment:
Can this section be structured in a similar way to the FWU SRA: https://github.com/ARM-software/psa-api/blob/80ea7319d0e86fae6d4328cf28ee3d188461ae69/doc/fwu/appendix/sra.rst?plain=1#L869

This splits the mitigations by Control-by-API, Transfer-to-implementation, and Transfer-to-application/other. It also provides a brief introduction to each summary table to explain this mitigation category.